src/main/java/org/opensearch/security/auditlog/impl/RequestResolver.java

```diff
@@ -381,7 +382,7 @@ private static void addIndicesSourceSafe(
         final String[] indices,
         final IndexNameExpressionResolver resolver,
         final ClusterService cs,
-        final XContentType xContentType,
+        final MediaType xContentType,
```

Comment:
Nit: Wdyt to rename the variable `xContentType` to `mediaType` for better readability?

Reply:
Hi @willyborankin, thats a good call for sure. I have changed them.